src/widgets/health-monitoring/HealthMonitoringTile.tsx

```diff
@@ -189,7 +189,8 @@ const SystemStatusTile = ({ data, properties }: { data: any; properties: any })
   const formatUptime = (uptime: number) => {
     const days = Math.floor(uptime / (60 * 60 * 24));
     const remainingHours = Math.floor((uptime % (60 * 60 * 24)) / 3600);
-    return t('info.uptimeFormat', { days: days, hours: remainingHours})
+    const remainingMinutes = Math.floor((uptime % 3600) / 60);
+    return t('info.uptimeFormat', { days: days, hours: remainingHours, minutes: remainingMinutes });
```

Comment:
I like that you added the minutes, it actually makes sense to have that info too.

This is only a suggestion. You didn't make the original function so you don't have to change it.
I just really don't like the way it's done here because it's legibility isn't great with all the numbers.
```TS
const time = dayjs.duration(uptime);
return t('info.uptimeFormat', { 
   days: Math.floor(time.asDays()),
   hours: time.hours(),
   minutes: time.minutes(), 
});
```

Reply:
Uhm yes, should i leave it so? Any Problems left or can it be than Approved?